ara/models.py

```diff
@@ -31,6 +32,30 @@ def mkuuid():
 
     return str(uuid.uuid4())
 
```

Comment:
I understand the DRY intended purpose here but I'm not sure this removes enough bloat to be worth it (zen of python)